morphic.js

```diff
@@ -7818,10 +7824,40 @@ MenuMorph.prototype.select = function (aMenuItem) {
     this.selection = aMenuItem;
 };
 
+MenuMorph.prototype.mouseLeave = function () {
+    if (this.parent instanceof MenuMorph) {
+        // this is a submenu so we vanish when lose focus
+		this.destroy();
+    }
+}
```

Comment:
Oh fine.  I point out that JSHint gives about 100 complaints, of which about five were mine.  A certain amount of the pot calling the kettle African-American going on here. :(